### ISSUE TITLE
The latest pbstarphase 0.11.2 freshly installed via conda failed to build the Json database from CPIC

### ISSUE THREAD

**Issue by davidyuyuan** [PacificBiosciences/pb-StarPhase#19]
Installed hifihla hiphase pbstarphase from bioconda:
```
#8 [production  2/13] RUN conda install -y -c bioconda hifihla hiphase pbstarphase
#8 14.83 
#8 14.83 ## Package Plan ##
#8 14.83 
#8 14.83   environment location: /opt/conda
#8 14.83 
#8 14.83   added / updated specs:
#8 14.83     - hifihla
#8 14.83     - hiphase
#8 14.83     - pbstarphase
#8 14.83 
#8 14.83 
#8 14.83 The following packages will be downloaded:
#8 14.83 
#8 14.83     package                    |            build
#8 14.83     ---------------------------|-----------------
#8 14.83     ca-certificates-2024.7.2   |       h06a4308_0         127 KB
#8 14.83     certifi-2024.7.4           |  py312h06a4308_0         159 KB
#8 14.83     conda-24.5.0               |  py312h06a4308_0         1.2 MB
#8 14.83     frozendict-2.4.2           |  py312h06a4308_0          36 KB
#8 14.83     hifihla-0.3.1              |       hdfd78af_0        13.0 MB  bioconda
#8 14.83     hiphase-1.4.2              |       h9ee0642_0         3.8 MB  bioconda
#8 14.83     openssl-3.0.14             |       h5eee18b_0         5.2 MB
#8 14.83     pbstarphase-0.11.2         |       h9ee0642_0         5.7 MB  bioconda
#8 14.83     ------------------------------------------------------------
#8 14.83                                            Total:        29.2 MB
#8 14.83 
#8 14.83 The following NEW packages will be INSTALLED:
#8 14.83 
#8 14.83   frozendict         pkgs/main/linux-64::frozendict-2.4.2-py312h06a4308_0 
#8 14.83   hifihla            bioconda/noarch::hifihla-0.3.1-hdfd78af_0 
#8 14.83   hiphase            bioconda/linux-64::hiphase-1.4.2-h9ee0642_0 
#8 14.83   pbstarphase        bioconda/linux-64::pbstarphase-0.11.2-h9ee0642_0 
#8 14.83 
#8 14.83 The following packages will be UPDATED:
#8 14.83 
#8 14.83   ca-certificates                      2024.3.11-h06a4308_0 --> 2024.7.2-h06a4308_0 
#8 14.83   certifi                          2024.2.2-py312h06a4308_0 --> 2024.7.4-py312h06a4308_0 
#8 14.83   conda                              24.3.0-py312h06a4308_0 --> 24.5.0-py312h06a4308_0 
#8 14.83   openssl                                 3.0.13-h7f8727e_0 --> 3.0.14-h5eee18b_0 
#8 14.83 
#8 14.83 
#8 14.83 
#8 14.83 Downloading and Extracting Packages: ...working... done
#8 14.83 Preparing transaction: ...working... done
#8 14.86 Verifying transaction: ...working... done
#8 15.60 Executing transaction: ...working... done
#8 DONE 20.4s
```
Building the CPIC database failed with error code 74:
```
#19 [production 11/13] RUN /opt/conda/bin/pbstarphase build --output-db "/home/jovyan/pgxdash/pgxtools/variants/cpic_json_db.json"
#19 0.619 [2024-07-15T14:03:13.271Z INFO  pbstarphase::cli::db_build] Output database: "/home/jovyan/pgxdash/pgxtools/variants/cpic_json_db.json"
#19 0.619 [2024-07-15T14:03:13.271Z INFO  pbstarphase::build_database] Starting CPIC API queries...
#19 0.619 [2024-07-15T14:03:13.271Z INFO  pbstarphase::build_database] 	Querying gene list via https://api.cpicpgx.org/v1/gene?url=not.eq.null&order=symbol
#19 1.143 [2024-07-15T14:03:13.794Z INFO  pbstarphase::build_database] 	Querying "all_genes" via https://api.cpicpgx.org/v1/allele_definition?select=*,%20allele_location_value(*,%20sequence_location(*))&order=name
#19 2.146 [2024-07-15T14:03:14.798Z INFO  pbstarphase::build_database] Starting HLA queries...
#19 2.147 [2024-07-15T14:03:14.798Z INFO  pbstarphase::build_database] 	Querying lastest HLA tag via https://api.github.com/repos/ANHIG/IMGTHLA/releases/latest
#19 2.325 [2024-07-15T14:03:14.976Z INFO  pbstarphase::build_database] Found latest HLA version: v3.57.0-alpha
#19 2.325 [2024-07-15T14:03:14.976Z INFO  pbstarphase::build_database] 	Querying HLA DNA sequences via https://raw.githubusercontent.com/ANHIG/IMGTHLA/v3.57.0-alpha/fasta/hla_gen.fasta
#19 2.461 [2024-07-15T14:03:15.112Z ERROR pbstarphase] Error while building CPIC database: Expected > at record start.
#19 ERROR: process "/bin/sh -c /opt/conda/bin/pbstarphase build --output-db \"${CPIC_JSON_DB}\"" did not complete successfully: exit code: 74
```

**Comment by davidyuyuan** [comment]
Thank you for the prompt response, Matt. Our last successful build was on pbstarphase 0.11.1 at 8 July 2024 at 09:36:52 GMT-4. Perphas, IMGT-HLA database at EBI got updated after that? Just FYI.

**Comment by holtjma** [comment]
Yes, with v3.57.0-alpha, they started .zip compressing the HLA sequences, which pb-StarPhase was not prepared for.

**Comment by holtjma** [comment]
@davidyuyuan This has been resolved with StarPhase v0.11.3, which can be found here: 
https://github.com/PacificBiosciences/pb-StarPhase/releases/tag/v0.11.3

It should appear on bioconda within a few hours.
